tools/travis-test.sh

```diff
@@ -23,7 +23,16 @@ setup_base()
   # install' also has the advantage that it tests that numpy is 'pip
   # install' compatible, see e.g. gh-2766...
 if [ -z "$USE_DEBUG" ]; then
-  $PIP install .
+  if [ -z "$IN_CHROOT" ]; then
+    $PIP install .
+  else
+    sysflags="$($PYTHON -c "from distutils import sysconfig; print (sysconfig.get_config_var('CFLAGS'))")"
+    # windows compilers have this requirement
```

Comment:
MSVC, its a copy paste from a few lines below which we use to avoid these issues poping up at the last minute when someone bothers to compile with it.